cli/ps.go

```diff
@@ -40,7 +44,9 @@ func (p *PsCommand) Init(c *Cli) {
 
 // addFlags adds flags for specific command.
 func (p *PsCommand) addFlags() {
-	// TODO: add flags here
+	flagSet := p.cmd.Flags()
+	flagSet.BoolVarP(&p.flagAll, "all", "a", false, "Show all containers (default shows just running)")
```

Comment:
We should add this parameter support in daemon side rather than only in client side. WDYT?